src/Fields.php

```diff
@@ -516,11 +524,11 @@ protected function wf_crm_get_fields($var = 'fields') {
           }
         }
       }
-      $all_tagsets = wf_crm_apivalues('tag', 'get', [
-        'return' => ['id', 'name', 'used_for'],
+      $all_tagsets = wf_crm_apivalues('tag', 'get', array(
+        'return' => array('id', 'name', 'used_for'),
         'is_tagset' => 1,
-        'parent_id' => ['IS NULL' => 1],
-      ]);
+        'parent_id' => array('IS NULL' => 1),
+      ));
```

Comment:
Looks like this part of the patch is accidentally undoing short array syntax.
IMO we should autoformat both 7.x and 8.x branches to replace all arrays with short syntax. That should make porting a little simpler hopefully.

Reply:
Yes - sorry about that. I've (re)fixed my PhpStorm settings so that should not be happening again. I like the idea of autoformating both 7.x and 8.x branches to replace all arrays with short syntax.